site/app/views/NavigationView.php

```diff
@@ -449,15 +449,19 @@ private function getSubmitButton(Gradeable $gradeable, $graded_gradeable, int $l
                 $title = "TA GRADE NOT AVAILABLE";
             }
         } else {
-            // This means either the user isn't on a team, or has no submissions yet
+            // This means either the user isn't on a team
             if ($gradeable->isTeamAssignment()) {
+                // access in the view... seems bad
+                $submit_any = $this->core->getAccess()->canI('gradeable.submit_any', ['gradeable' => $gradeable]);
```

Comment:
Would highly recommend moving this to the controller and passing in a var eg `$show_submit_no_team`